### PR TITLE
Scanning Barcode issue coming for the second instance is  resolved

### PR DESCRIPTION
**Pull request by shreyas098**
### What does this PR do?
Scanning the same barcode twice or more makes it viewable

BUG: Scanning the same barcode twice makes inventory no. unviewable 

NOTE: The observer present was triggering only once based based on query params for which another observer was added to have a check on search Text which resolved the issue.

### main change : scannedText

### CODE REVIEW

File: app/controllers/items/index.js [hunk 1]
```diff
@@ -33,6 +33,18 @@ export default Ember.Controller.extend(SearchMixin, {
     }
   }),
 
+  scannedText: Ember.observer("searchText", function() {
+    const searchInput = this.get("searchText") || "";
+    const sanitizeString = this.sanitizeString(searchInput);
+    //if (sanitizeString) {
```
Comment (abhinav098) [inline app/controllers/items/index.js]: @shreyas098 , why is this commented? If this is not required please remove it.

Keeping a check and then setting the `searchInput` is better.

Please let me know your thoughts.

File: app/controllers/orders/detail.js [hunk 2]
```diff
@@ -37,12 +37,18 @@ export default GoodcityController.extend(SearchMixin, {
...
     return (
       ["client_summary", "contact_summary"].indexOf(this.get("tabName")) >= 0
     );
   }),
 
+  highlightSelectedTabGoods: Ember.computed("tabName", function() {
+    return (
+      ["active_items", "requested_items"].indexOf(this.get("tabName")) >= 0
+    );
+  }),
```
Comment (abhinav098) [inline app/controllers/orders/detail.js]: As far as I know, this fix is for a separate ticket, please create a new branch from master and move this code to that branch @shreyas098 . We should not merge two tickets together while fixing issues or any other thing for that matter.

File: app/templates/orders/_order_tabs.hbs [hunk 3]
```diff
@@ -3,14 +3,15 @@
     <div class="small-12 columns tab_row">
       <dl class="tabs" data-tab>
         <dd class="small-3 medium-3 columns text-center
-          {{if highlightSelectedTab 'active'}}">
+          {{if highlightSelectedTabSummary 'active'}}">
```
Comment (abhinav098) [inline app/templates/orders/_order_tabs.hbs]: same as above.

File: app/templates/orders/_order_tabs.hbs [hunk 4]
```diff
@@ -3,14 +3,15 @@
...
           {{#link-to 'orders.contact_summary' model.id replace=true}}
           <i>{{fa-icon 'id-card'}}</i>
           <div>{{t "order_tabs.summary"}}</div>
           {{/link-to}}
         </dd>
 
-        <dd class="small-3 medium-3 columns text-center {{if (is-equal tabName 'active_items') 'active'}}">
+        <dd class="small-3 medium-3 columns text-center
+        {{if highlightSelectedTabGoods 'active'}}">
```
Comment (abhinav098) [inline app/templates/orders/_order_tabs.hbs]: same as above.

File: app/controllers/orders/detail.js [hunk 5]
```diff
@@ -37,12 +37,18 @@ export default GoodcityController.extend(SearchMixin, {
       .get("lastObject");
   }),
 
-  highlightSelectedTab: Ember.computed("tabName", function() {
+  highlightSelectedTabSummary: Ember.computed("tabName", function() {
```
Comment (abhinav098) [inline app/controllers/orders/detail.js]: Kindly move these fixes to a separate branch.

**Comment by shreyas098** [comment]
Two Branches were merged mistakenly so creating new pull request for the same individual branches

**Pull request closed without merging**